test/common/assertSnapshot.js

```diff
@@ -51,11 +50,15 @@ async function assertSnapshot(actual, filename = process.argv[1]) {
  *
  * @param {string} filename
  * @param {function(string): string} [transform]
+ * @param {object} [options] - options to pass to spawnPromisified
```

Comment:
I wouldn’t really consider these options passed along to `spawnPromisified`.

```suggestion
 * @param {object} [options] - control how the child process is spawned
```